doc/content/guides/connecting-gateways/kerlinkwirnet/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Kerlink Wirnet Station"
+description: ""
+weight: 1
+---
+
+Kerlink Wirnet Station is a LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.kerlink.com/product/wirnet-station/). This page guides you to connect it to The Things Stack.
+
+## Prerequisites
+
+1. User account on The Things Stack with rights to create Gateways and API Keys.
+2. Kerlink Wirnet Station:
+  1. For configuration via provisioning script: Common Packet Forwarder installed and enabled and/or gateway running firmware version `3.0` or higher.
+  2. For manual configuration: Common Packet Forwarder installed and enabled
+
+> NOTE: Minimum CPF version tested is `v1.1.6`
+
+## Registration
+
+Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}). Choose a **Gateway ID** that is at least 6 characters in length. An **EUI** is not necessary.
```

Comment:
1. Remove "at least 6 characters in length"; it's not an accurate description of our ID validation
2. It is very much recommended to enter an EUI

Reply:
That is copy-pasted from TTG guide, will fix both

Reply:
TTKG does not have an EUI so we recommend against entering one, because they may take a valid EUI.

Kerlinks do have an EUI instead.

> What's leading here - user config or gateway default?

Leading for what?

Reply:
TTKG does not have an EUI so we recommend against entering one, because they may take a valid EUI.

Kerlinks do have an EUI instead.

> What's leading here - user config or gateway default?

Leading for what?

Reply:
Gateway configuration allows the user to configure a EUI.
I suppose in this case we want the default gateway EUI to be leading instead of letting the user choose it and configure it.

Reply:
No, the ID and EUI are two separate things. IDs allow for something that's human friendly, while the EUI is to keep things unique across environments/tenants

Reply:
I notice that this part is identical to the one in [the merged PR regarding the  Tektelic Kona Micro IoT LoRaWAN Gateway](https://github.com/TheThingsNetwork/lorawan-stack/pull/1742/files#diff-448b6fce7e76f5e4db58fb2a74246b4eR14-R16). It is also most likely going to be identical with the guides for the rest of the gateways.

Would it pay to have that extracted in a single page with a title like "common steps for all gateways"? It would probably help a lot with maintaining the documentation and keeping it concise. The one dowside I can think of is that we then require the user to browse through multiple pages in order to setup his gateway, and that might scare him away or something.

Thoughts?

Reply:
This also crossed my mind before, but at this point the only shared part is this sentence:
> Create a gateway by following the instructions for the [Console]({{< ref "/guides/getting-started/console#create-gateway" >}}) or the [CLI]({{< ref "/guides/getting-started/cli#create-gateway" >}}).

The rest of the guides is actually different. And I don't think this sentence can (or should) be extracted anywhere.